server/db.ts

```diff
@@ -0,0 +1,14 @@
+import mysql from 'mysql2';
+import dotenv from 'dotenv';
+
+dotenv.config() //.env
+
+
+export const connection = mysql.createConnection({
+    host: process.env.DB_HOST,
+    user: process.env.DB_USER,
+    password: process.env.DB_PASS,
+    database: process.env.DB_NAME
+}).on('error', function (err: any) {
+    console.error('mysql failed', err)
+})
```

Comment:
lets remove this on(error) line and let it break if this throws error - by having this when db connection failed, your app still run